### PR TITLE
fix: dep update

### PR DESCRIPTION
**Pull request by simlu**

### CODE REVIEW

File: src/modules/request-recorder.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ module.exports = (opts) => {
             });
             await new Promise((resolve) => {
               options.protocol = `${protocol}:`;
-              const r = { http, https }[protocol].request(options, (response) => {
+              const r = { http, https: http }[protocol].request(options, (response) => {
```
Comment (krackerman) [inline src/modules/request-recorder.js:105]: Why are we doing this?

Comment (simlu) [reply src/modules/request-recorder.js:105]: haha good question

Comment (simlu) [reply src/modules/request-recorder.js:105]: it's so that we can support other protocols easily in the future

File: test/util/desc.spec.js [hunk 2]
```diff
@@ -184,11 +188,10 @@ describe('Testing { describe }', () => {
...
     it('Test capture throws when no error captured', async ({ capture }) => {
-      try {
-        await capture(() => {});
-      } catch (e) {
-        expect(String(e)).to.equal('AssertionError [ERR_ASSERTION]: expected [Function] to throw an error');
-      }
+      const err = await new Promise((resolve) => {
+        capture(() => {}).catch((e) => resolve(e));
+      });
+      expect(String(err)).to.equal('AssertionError [ERR_ASSERTION]: expected [Function] to throw an error');
```
Comment (krackerman) [inline test/util/desc.spec.js:194]: Was there coverage missing here?

Comment (simlu) [reply test/util/desc.spec.js:194]: yep, that's why its done this way now

**Review by krackerman — APPROVED** [review]

**Pull request merged**